runtime/gc_glue_java/MarkingDelegate.cpp

```diff
@@ -315,9 +317,11 @@ MM_MarkingDelegate::scanContinuationNativeSlots(MM_EnvironmentBase *env, omrobje
 		stackFrameClassWalkNeeded = isDynamicClassUnloadingEnabled();
 #endif /* J9VM_GC_DYNAMIC_CLASS_UNLOADING */
 
+#if JAVA_SPEC_VERSION >= 19
```

Comment:
I think this line should be before line 315, else there's an unused variable warning (error).

Reply:
true, although the best is just to put the whole method (scanContinuationNativeSlots) under the build flag